app/src/main/java/com/example/cm/SplashScreenActivity.java

```diff
@@ -30,8 +39,40 @@ protected void onCreate(Bundle savedInstanceState) {
         if (firebaseUser != null) {
             intent = new Intent(this, MainActivity.class);
         } else {
-            intent = new Intent(this, LoginActivity.class);
+            intent = new Intent(this, OnboardingActivity.class);
         }
         startActivity(intent);
+      
+        setContentView(R.layout.activity_splash_screen);
+
+        setupUI();
+    }
+    
+    private void setupUI() {
+        splashImage = findViewById(R.id.splash_img);
+        splashAnim = AnimationUtils.loadAnimation(this, R.anim.bottom_anim);
+
+        splashImage.setAnimation(splashAnim);
+
+        new Handler().postDelayed(() -> {
+            onBoardingSP = getSharedPreferences("onBoarding", MODE_PRIVATE);
+
+            boolean isFirstTime = onBoardingSP.getBoolean("firstTime", true);
+
+            if (isFirstTime) {
+
+                SharedPreferences.Editor editor = onBoardingSP.edit();
+                editor.putBoolean("firstTime", false);
+                editor.commit();
```

Comment:
according to [article](https://stackoverflow.com/questions/5960678/whats-the-difference-between-commit-and-apply-in-sharedpreferences) you should use apply